### DISCUSSION TITLE
Failsafes for purchasing a server when the node doesn't have enough resources

### DISCUSSION THREAD

**Discussion by purejosh** [Paymenter/Paymenter#439] [Ideas]
Similar to #421, but worded better:

A user should either get a warning, or the add to cart button should be grayed out, if the nodes allowed for that package don't contain enough resources to satisfy the purchase. For example, if someone's buying a 12GB RAM server, and the nodes in that location only have 4GB and 8GB, it should have some sort of intervention for that transaction. 

An idea came up in the Discord server to update server resources every purchase, or daily using a cronjob, which is a great idea. Another idea was to allow the purchase to go through, but put the server in "pending creation" status or something similar, where it would require intervention from the server host to correct the deployment. 

Thanks for your consideration.

**Comment by NightmareFox** [comment]
Yea i agree with that.
